src/main/java/Deadline.java

```diff
@@ -0,0 +1,30 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Creates a Deadline task with a date.
+ */
+public class Deadline extends Task {
+    protected LocalDate by;
+
+    /**
+     * Creates a new Deadline.
+     * @param description Description of the task.
+     * @param by Date of the deadline.
+     */
+    public Deadline(String description, LocalDate by) {
+        super(description);
```

Comment:
Maybe it will be better if you name your parameter using a noun such as "date"!!